alembic/templates/generic/env.py

```diff
@@ -1,10 +1,15 @@
+import os
 from logging.config import fileConfig
 
+from dotenv import load_dotenv
```

Comment:
I don't think the general template should import a 3rd party dependency

Reply:
fair